packages/postcss/README.md

```diff
@@ -79,7 +79,32 @@ For more information on that topic, please have a look at [the documentation of
 
 #### Preset Options
 
-This preset has no preset configuration options.
+| Name | Type | Default | Required | Description |
+| --- | --- | --- | --- | --- |
```

Comment:
Can you please run `yarn fmt` I think prettier will format this file differently ;)

Reply:
No, this is how Prettier likes it to be.